client/login/controller.js

```diff
@@ -11,12 +11,15 @@ import MagicLogin from './magic-login';
 
 export default {
 	login( context, next ) {
-		context.renderCacheKey = 'login';
-		context.primary = <WPLogin twoFactorAuthType={ context.params.twoFactorAuthType } />;
+		const { lang, path, params } = context;
+
+		context.renderCacheKey = `login:${ lang }`;
```

Comment:
We no longer need this cache key since https://github.com/Automattic/wp-calypso/pull/13339. That's something that is going to be addressed in https://github.com/Automattic/wp-calypso/pull/14924 by the way.